### DISCUSSION TITLE
DAVx5 Request - jtxBoard vs Task.org Integration by server account, not globally

### DISCUSSION THREAD

**Discussion by thomascrisan** [TechbeeAT/jtxBoard#244] [General]
I currently use Tasks.org with DAVx5 and would also like to use jtxBoard with a different CalDAV server.  At the moment, the setting in DAVx5 for Tasks.org or jtxBoard integration is at the application level, not by account.  I put in a suggestion that the integration setting for Tasks.org or jtxBoard be set by account, not globally in the application.

[
https://github.com/bitfireAT/davx5-ose/discussions/224globaly](https://github.com/bitfireAT/davx5-ose/discussions/224globaly)

If you think this is a good idea, please vote for this idea on the DAVx5 discussion.

**Comment by patrickunterwegs** [comment]
Hey @thomascrisan , thanks for opening the discussion! It would definitely be possible, at least to choose a journals/notes provider and tasks provider independently. It would require some architectural changes in DAVx5 - so the change first needs to be done there. 

For the discussion itself: 
It is still not that trivial, if you consider that jtx Board can also have subtasks in journals or notes. If you consider using jtx Board only for notes and journals, would the remote collection then be forced to only support VJOURNAL? If you consider still having subtasks, they would also appear in Tasks.org. After all DAVx5 would need to take care that also the apps are kept in sync (as there is no common storage for the data). This could then lead to inconsistencies and conflicts. 
So if this change is done, it must be very well thought through!
